main/src/main/java/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -205,7 +205,8 @@ public class CacheDetailActivity extends TabbedViewPagerActivity
     public static final String STATE_PAGE_INDEX = "cgeo.geocaching.pageIndex";
     public static final String STATE_IMAGE_GALLERY = "cgeo.geocaching.imageGallery";
     public static final String STATE_DESCRIPTION_STYLE = "cgeo.geocaching.descriptionStyle";
-    public static final String STATE_SPEECHSERVICE_RUNNING = "cgeo.geocachig.speechServiceRunning";
+    public static final String STATE_SPEECHSERVICE_RUNNING = "cgeo.geocaching.speechServiceRunning";
+    public static final String STATE_TRANSLATION_LANGUAGE_SOURCE = "cgeo.geocaching.translation.languageSource";
```

Comment:
Is there any naming-convention for the save-states?

Reply:
Not as far as I know